src/ol/proj/proj.js

```diff
@@ -120,6 +120,9 @@ ol.proj.Projection = function(options) {
    * @type {boolean}
    */
   this.global_ = goog.isDef(options.global) ? options.global : false;
+  if (this.global_) {
+    goog.asserts.assert(!goog.isNull(this.extent_));
```

Comment:
Makes sense. This means I have to remove this assertion and check for extent available elsewhere before doing any wrapping. I can do that.